code/game/objects/items/weapons/defib.dm

```diff
@@ -1,5 +1,14 @@
-//backpack item
-
+/* CONTENTS
```

Comment:
I like the added organization here, but please do it in a separate PR. It's much more difficult to pick out changes when you move code and change it at the same time.

Reply:
I've unsorted everything. All the reorganisation is stored on a separate branch for each file for later deployment.